tests/test_phonon.py

```diff
@@ -1,22 +1,34 @@
 """Tests for PhononCalc class"""
+
 from __future__ import annotations
 
+import os
 from typing import TYPE_CHECKING
 
 import pytest
 
 from matcalc.phonon import PhononCalc
 
 if TYPE_CHECKING:
+    from pathlib import Path
+
     from matgl.ext.ase import M3GNetCalculator
     from pymatgen.core import Structure
 
 
-def test_phonon_calc(Li2O: Structure, M3GNetCalc: M3GNetCalculator) -> None:
+def test_phonon_calc(Li2O: Structure, M3GNetCalc: M3GNetCalculator, tmp_path: Path) -> None:
     """Tests for PhononCalc class"""
     # Note that the fmax is probably too high. This is for testing purposes only.
+
+    # change dir to tmp_path
+    phonon_yaml = tmp_path / "phonon.yaml"
     phonon_calc = PhononCalc(
-        M3GNetCalc, supercell_matrix=((2, 0, 0), (0, 2, 0), (0, 0, 2)), fmax=0.1, t_step=50, t_max=1000
+        M3GNetCalc,
+        supercell_matrix=((2, 0, 0), (0, 2, 0), (0, 0, 2)),
+        fmax=0.1,
+        t_step=50,
+        t_max=1000,
+        write_phonon=phonon_yaml,
```

Comment:
i added a test for `write_phonon`. we need to cover the other new keywords as well.